openquake/hazardlib/gsim/base.py

```diff
@@ -322,6 +323,11 @@ def __init__(self, **kwargs):
             msg = ('%s is experimental and may change in future versions - '
                    'the user is liable for their application') % cls.__name__
             warnings.warn(msg, ExperimentalWarning)
+        if cls.non_general:
```

Comment:
Honestly, I do not like the name `non_general`, is too vague. Also, it does not look different enough from `experimental` to me.

Reply:
A name along the lines of "specific_case" or "adapted" might be better. We wanted to add a warning to indicate a model has been adapted/used for a specific case but may not apply elsewhere. To me this is different from experimental. Not sure the others agree.

Reply:
I like the name "adapted"

Reply:
I also like "adapted". Are you otherwise okay with the text in the warning message?

Reply:
Yes, the text is fine.